pages/transcripts/react-native-core-team.md

```diff
@@ -0,0 +1,682 @@
+---
+title: React Native Core Team
```

Comment:
I wonder if it would be helpful to put (May 2019) here since we have more than one RN team Q&A? Might be better to just eventually add dates to the transcription list though.

Reply:
I agree. There's one more React Native Core team Q&A after this so far. I'll add it there too.

Reply:
Added (May 2019) in the title.

Reply:
Thanks!